### PR TITLE
bump to 0.6.0-S

### PR DESCRIPTION
**Pull request by glorysdj**
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/1421)

### CODE REVIEW

File: apps/model-inference-examples/model-inference-flink/pom.xml [hunk 1]
```diff
@@ -42,8 +42,8 @@
         </dependency>
         <dependency>
             <groupId>com.intel.analytics.zoo</groupId>
-            <artifactId>analytics-zoo-bigdl_0.7.2-spark_2.4.0</artifactId>
-            <version>0.5.0-SNAPSHOT</version>
+            <artifactId>analytics-zoo-bigdl_0.8.0-spark_2.4.0</artifactId>
```
Comment (hkvision) [inline apps/model-inference-examples/model-inference-flink/pom.xml]: change to 2.4.3?

Comment (glorysdj) [reply apps/model-inference-examples/model-inference-flink/pom.xml]: ok

**Comment by glorysdj** [comment]
> Python version?

what do you mean? 

> Python version?

forget to commit

**Comment by hkvision** [comment]
LGTM

**Pull request merged**